recipes/xstatic-bootstrap/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "XStatic-Bootstrap" %}
+{% set version = "4.1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1800e6bb5fc687604d8a893eee8c7882d800a6f3d6721799016f99a47d1dac0f
+
+build:
+  number: 0
```

Comment:
This recipe can be noarch: python

Reply:
Quick question @ReimarBauer. Did you create this recipe with [grayskull](https://pypi.org/project/grayskull/)? It should've done the `noarch` suggestion for you there.